recipes/conda-pip-minimal/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "conda-pip-minimal" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/conda_pip_minimal-{{ version }}.tar.gz
+  sha256: b27138eb96ffa1c57c7d1c2a423b373baaa22e70ce1086fe8538dbe276566867
+
+build:
+  entry_points:
+    - conda-cpm = conda_pip_minimal.cli:app
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - hatchling
+    - pip
+  run:
+    - python >=3.8
+    - typer >=0.6.1,<0.7.dev0
+    - semver >=3.0.0
+    - more-itertools
+    - trio >=0.22.0,<0.23.dev0
+    - loguru
+    - importlib-metadata >=5.0.0,<5.1.dev0
+    - ruamel_yaml >=0.17.21,<0.18.dev0
+
+test:
+  imports:
+    - conda_pip_minimal
+  requires:
+    - pip
+    - pytest
+    - conda-cpm --help
+  source_files:
+    - tests
```

Comment:
If you aren't running the test suite, I don't think you need the `source_files`. Same goes with `pytest` under `requires`

Reply:
I would like to run the test suite and plan to add more tests in the next patch. I wanted to get this PR merged first though. Should I keep this as is, remove and add next release or add a simple pytest now?